kola/harness.go

```diff
@@ -460,19 +460,21 @@ func SetupOutputDir(outputDir, platform string) (string, error) {
 	}
 
 	if defaulted {
+		tempLinkPath := filepath.Join(outputDir, "latest")
 		linkPath := filepath.Join(defaultBaseDirName, platform+"-latest")
+		// safety check
```

Comment:
Maybe something more descriptive like `do not clobber existing files that are not symlinks` ?